### ISSUE TITLE
spotify-adblock is not blocking ads

### ISSUE THREAD

**Issue by inkryougan** [abba23/spotify-adblock#92]
Whenever I run spotify in the terminal, I get this output:

```
spotify: /usr/lib/libcurl-gnutls.so.4: no version information available (required by spotify)
/opt/spotify/spotify: /usr/lib/libcurl-gnutls.so.4: no version information available (required by /opt/spotify/spotify)
/opt/spotify/spotify: /usr/lib/libcurl-gnutls.so.4: no version information available (required by /opt/spotify/spotify)
/proc/self/exe: /usr/lib/libcurl-gnutls.so.4: no version information available (required by /proc/self/exe)
```

I use Arch Linux and I installed spotify-dev using the AUR package. I get no banner ads because I use another AUR package [spotify-remove-ad-banner](https://aur.archlinux.org/packages/spotify-remove-ad-banner). Am I doing something wrong? What should I do to remove ads?

Here is an image of the ad: [https://imgur.com/a/xaBVahR](https://imgur.com/a/xaBVahR)

**Comment by abba23** [comment]
Did you launch Spotify as specified in the readme?

**Comment by inkryougan** [comment]
> Did you launch Spotify as specified in the readme?

No. At first, I didn't. I checked the readme and used that command, it still isn't working.

**Comment by abba23** [comment]
What's the output when you use the correct command then?

**Comment by pupperemeritus** [comment]
I also am having a similar issue. Spotify-Adblock nor  regular spotify dont launch from .desktop files unless I use the launch command. but they do launch from terminal. and the adblock command isn't blocking ads anymore.  It was working until yesterday iirc. Only since today this issue started popping up.

**Comment by abba23** [comment]
@pupperemeritus: See #3.

**Comment by pupperemeritus** [comment]
Alright it is working now. Thank you mate. Ad blocking also is working now.
